macros/core/src/program/mod.rs

```diff
@@ -332,8 +349,21 @@ fn generate_handle<'a>(
         "Unexpected service",
     ));
 
+    let mut args = Vec::with_capacity(2);
+    if let Some(handle_reply) = program_args.handle_reply() {
+        args.push(quote!(handle_reply = #handle_reply));
+    }
+    if let Some(handle_signal) = program_args.handle_signal() {
+        args.push(quote!(handle_signal = #handle_signal));
+    }
+    let async_main_args = if args.is_empty() {
+        quote!()
```

Comment:
Doesn't make much sense, the below quote will result in nothing if `args` is empty

Reply:
it is for generating `#[gstd::async_main]` without `()`.

Reply:
Does it matter? Can't it have empty braces?

Reply:
you can) just to minimize changes in insta files with this PR